contracts/protocol/libraries/logic/GenericLogic.sol

```diff
@@ -152,13 +160,13 @@ library GenericLogic {
     DataTypes.UserConfigurationMap memory userConfig,
     mapping(uint256 => address) storage reserves,
     uint256 reservesCount,
-    address oracle
+    Params memory params
   ) internal view returns (uint256, uint256, uint256, uint256, uint256) {
     CalculateUserAccountDataVars memory vars;
 
-    if (userConfig.isEmpty()) {
-      return (0, 0, 0, 0, uint256(-1));
-    }
+    // if (userConfig.isEmpty()) {
```

Comment:
why did we remove this check?

Reply:
This will fail when user has only ithaca collateral as the userConfig will be empty then, so in order to allow user to borrow with just ithaca collateral, this is removed

Reply:
understood